server/src/users/users.controller.ts

```diff
@@ -18,7 +18,7 @@ export class UsersController {
     return user;
   }
 
-  @UseGuards(JwtAuthGuard)
+  // @UseGuards(JwtAuthGuard)
```

Comment:
let's remove this endpoint altogether as well
make sure nothing hits it first but i don't see a need for it right now